Library/Formula/gnuradio-osmosdr.rb

```diff
@@ -0,0 +1,87 @@
+class GnuradioOsmosdr < Formula
+  desc "osmocom Gnu Radio Blocks"
+  homepage "http://sdr.osmocom.org/trac/wiki/GrOsmoSDR"
+  url "http://cgit.osmocom.org/gr-osmosdr/snapshot/gr-osmosdr-0.1.4.zip"
+  sha256 "a5c439f275cf11965d2fc67957e1d221b4f985ff5d87cdc9760522ba8319215d"
+
+  option "without-python", "Don't build gnuradio's python-support component"
+
+  patch :DATA
+
+  option :universal
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "boost"
+  depends_on "gnuradio"
+
+  if build.with? "python"
+    depends_on "swig" => :build
+  end
+
+  depends_on "hackrf" => :recommended
+  depends_on "librtlsdr" => :recommended
+  depends_on "uhd" => :recommended
+
+  resource "Markdown" do
+    url "https://pypi.python.org/packages/source/M/Markdown/Markdown-2.4.tar.gz"
+    sha256 "b8370fce4fbcd6b68b6b36c0fb0f4ec24d6ba37ea22988740f4701536611f1ae"
+  end
+
+  resource "Cheetah" do
+    url "https://pypi.python.org/packages/source/C/Cheetah/Cheetah-2.4.4.tar.gz"
+    sha256 "be308229f0c1e5e5af4f27d7ee06d90bb19e6af3059794e5fd536a6f29a9b550"
+  end
+
+  def install
+    args = std_cmake_args
+
+    if build.universal?
+      ENV.universal_binary
+      args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
+    end
+
+    ENV["CHEETAH_INSTALL_WITHOUT_SETUPTOOLS"] = "1"
+    ENV.prepend_create_path "PYTHONPATH", libexec+"vendor/lib/python2.7/site-packages"
+
+    res = %w[Markdown Cheetah]
+    res.each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    args << "-DENABLE_DEFAULT=OFF"
+    enabled_componenets = %w[fcd file rtl_tcp rfspace]
+    enabled_componenets += %w[python] if build.with? "python"
+    enabled_componenets += %w[rtl] if build.with? "librtlsdr"
+    enabled_componenets += %w[uhd] if build.with? "uhd"
+    enabled_componenets += %w[hackrf] if build.with? "hackrf"
+    enabled_componenets.each do |c|
+      args << "-DENABLE_#{c.upcase.split("-").join("_")}=ON"
```

Comment:
This `.split("-").join("_")` seems unnecessary, there isn’t any element with an hyphen.